package.json

```diff
@@ -22,6 +22,10 @@
     "@angular/platform-browser": "^17.3.0",
     "@angular/platform-browser-dynamic": "^17.3.0",
     "@angular/router": "^17.3.0",
+    "@ngrx/effects": "^18.0.0",
+    "@ngrx/entity": "^18.0.0",
```

Comment:
I don't see this being used

Reply:
No idea, came along when I installed ngrx 👀